app/Console/Commands/UpdatePostsWithActionIds.php

```diff
@@ -3,7 +3,6 @@
 namespace Rogue\Console\Commands;
 
 use DB;
-use Rogue\Models\Post;
```

Comment:
🤷‍♂ styleci cried about this -- It didn't seem to be used in this file so felt safe to remove!

Reply:
agreed that this is safe to remove! I wonder why styleci didn't catch it before 🤔